lib/subspace/commands/init.rb

```diff
@@ -6,13 +6,19 @@ def initialize(args, options)
     options.default env: "dev"
 
     @env = options.env
-    @template = options.template || "dev"
+    @template = options.template
 
     if options.ansibe.nil? && options.terraform.nil?
       # They didn't pass in any options (subspace init) so just do both
       options.ansible = true
       options.terraform = true
     end
+
+    if @template.nil? && options.terraform == true
+      say "Please specify a terraform template, such as 'workhorse' or 'oxenwagen': subspace init --env #{env} --template workhorse"
```

Comment:
Should we prompt for this instead?  We started doing that for the aws creds and I kindof like that workflow

Reply:
ooh probably

Reply:
i did it